guidelines/2016-03-08-github-guidelines.md

```diff
@@ -0,0 +1,115 @@
+# GitHub Guidelines
+
+`Editor note: This is the first draft of the GitHub guidelines for Origami.`
+
+
+## How to contribute to Origami
+
+Firstly, thank you for deciding to contribute to Origami! Please find the guidelines for creating issues or pull requests listed below.
+
+
+### Issues
+
+The first steps to undertake before creating an issue of any type on the repository is to search through the repository's Open/Closed issues to see whenever there's a ticket that replicates the same issue you are having and if there is a fix in progress or has been fixed. If there is no same issue that you are experiencing, then open a new ticket on the affected repository.
```

Comment:
Possibly also remove `to undertake` from the start, `The first steps before creating...` reads ok